code/modules/surgery/limb_manipulation.dm

```diff
@@ -123,6 +125,7 @@
 		if(istype(I))
 			I.state_flags = ORGAN_AUGMENTABLE
 			I.update_organ_icon()
+			target.regenerate_icons()
```

Comment:
I think the three instances of "target.regenerate_icons()", as well as the lines above and below can be just put once after all of this, no? That way there's less repetition